### ISSUE TITLE
Good news coming with Jasmine 3 🎉

### ISSUE THREAD

**Issue by yjaaidi** [Evilweed/protractor-beautiful-reporter#39]
Hi,

Once jasmine 3 will be ready and adopted, we won't need the `asyncFlow` hack to wait for the screenshots to be taken before moving to the next test etc...

https://github.com/jasmine/jasmine/issues/842

**Comment by Evilweed** [comment]
@yjaaidi wohoo! :)

**Comment by Evilweed** [comment]
#https://github.com/angular/protractor/issues/4788

**Comment by Evilweed** [comment]
Could be done after protractor/issues/4788

**Comment by miller45** [comment]
@Evilweed you are right as long as people use protractor 5.* will have to support the jasmine 2 way. It just tried it out. The screenshots are not in the right moment when p6 is used even with jasmine 3 as the developers framework,
